[2024-02-21T07:37:49.384+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-01-29T00:00:00+00:00 [queued]>
[2024-02-21T07:37:49.428+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-01-29T00:00:00+00:00 [queued]>
[2024-02-21T07:37:49.428+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-21T07:37:49.488+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_mail> on 2024-01-29 00:00:00+00:00
[2024-02-21T07:37:49.510+0000] {standard_task_runner.py:60} INFO - Started process 71 to run task
[2024-02-21T07:37:49.517+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'extract_email', 'get_mail', 'scheduled__2024-01-29T00:00:00+00:00', '--job-id', '61', '--raw', '--subdir', 'DAGS_FOLDER/extract_email.py', '--cfg-path', '/tmp/tmpfj7d4jy0']
[2024-02-21T07:37:49.522+0000] {standard_task_runner.py:88} INFO - Job 61: Subtask get_mail
[2024-02-21T07:37:49.736+0000] {task_command.py:423} INFO - Running <TaskInstance: extract_email.get_mail scheduled__2024-01-29T00:00:00+00:00 [running]> on host 893865381b0d
[2024-02-21T07:37:50.074+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_email' AIRFLOW_CTX_TASK_ID='get_mail' AIRFLOW_CTX_EXECUTION_DATE='2024-01-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-29T00:00:00+00:00'
[2024-02-21T07:37:50.076+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 196, in execute
    context_merge(context, self.op_kwargs, templates_dict=self.templates_dict)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/context.py", line 261, in context_merge
    context.update(*args, **kwargs)
  File "/usr/local/lib/python3.8/_collections_abc.py", line 837, in update
    for key, value in other:
ValueError: too many values to unpack (expected 2)
[2024-02-21T07:37:50.090+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=extract_email, task_id=get_mail, execution_date=20240129T000000, start_date=20240221T073749, end_date=20240221T073750
[2024-02-21T07:37:50.105+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 61 for task get_mail (too many values to unpack (expected 2); 71)
[2024-02-21T07:37:50.155+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-02-21T07:37:50.187+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-02-21T07:59:30.888+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-01-29T00:00:00+00:00 [queued]>
[2024-02-21T07:59:30.896+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-01-29T00:00:00+00:00 [queued]>
[2024-02-21T07:59:30.896+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-21T07:59:30.904+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_mail> on 2024-01-29 00:00:00+00:00
[2024-02-21T07:59:30.908+0000] {standard_task_runner.py:60} INFO - Started process 353 to run task
[2024-02-21T07:59:30.910+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'extract_email', 'get_mail', 'scheduled__2024-01-29T00:00:00+00:00', '--job-id', '124', '--raw', '--subdir', 'DAGS_FOLDER/extract_email.py', '--cfg-path', '/tmp/tmpqqg4r4o1']
[2024-02-21T07:59:30.912+0000] {standard_task_runner.py:88} INFO - Job 124: Subtask get_mail
[2024-02-21T07:59:30.946+0000] {task_command.py:423} INFO - Running <TaskInstance: extract_email.get_mail scheduled__2024-01-29T00:00:00+00:00 [running]> on host 9bab6a0db245
[2024-02-21T07:59:31.010+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_email' AIRFLOW_CTX_TASK_ID='get_mail' AIRFLOW_CTX_EXECUTION_DATE='2024-01-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-29T00:00:00+00:00'
[2024-02-21T07:59:33.558+0000] {ImapSmtp.py:839} INFO - IMAP search: 'gmail:'Message from RPA Python' after:15-12-2023'
[2024-02-21T07:59:37.105+0000] {logging_mixin.py:188} INFO - Message from RPA Python0
[2024-02-21T07:59:37.107+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-122.txt')
[2024-02-21T07:59:37.108+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:37.108+0000] {logging_mixin.py:188} INFO - Message from RPA Python1
[2024-02-21T07:59:37.109+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-123.txt')
[2024-02-21T07:59:37.110+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:37.110+0000] {logging_mixin.py:188} INFO - Message from RPA Python2
[2024-02-21T07:59:37.112+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-124.txt')
[2024-02-21T07:59:37.112+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:37.112+0000] {logging_mixin.py:188} INFO - Message from RPA Python3
[2024-02-21T07:59:37.113+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-125.txt')
[2024-02-21T07:59:37.114+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:37.114+0000] {logging_mixin.py:188} INFO - Message from RPA Python4
[2024-02-21T07:59:37.115+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-126.txt')
[2024-02-21T07:59:37.116+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:37.116+0000] {logging_mixin.py:188} INFO - Message from RPA Python5
[2024-02-21T07:59:37.118+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-127.txt')
[2024-02-21T07:59:37.118+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:37.118+0000] {logging_mixin.py:188} INFO - Message from RPA Python6
[2024-02-21T07:59:37.120+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-128.txt')
[2024-02-21T07:59:37.120+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:37.120+0000] {logging_mixin.py:188} INFO - Message from RPA Python7
[2024-02-21T07:59:37.122+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-129.txt')
[2024-02-21T07:59:37.122+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:37.122+0000] {logging_mixin.py:188} INFO - Message from RPA Python8
[2024-02-21T07:59:37.124+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-130.txt')
[2024-02-21T07:59:37.124+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:37.124+0000] {logging_mixin.py:188} INFO - Message from RPA Python9
[2024-02-21T07:59:37.126+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-131.txt')
[2024-02-21T07:59:37.126+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:37.722+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-21T07:59:37.729+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=extract_email, task_id=get_mail, execution_date=20240129T000000, start_date=20240221T075930, end_date=20240221T075937
[2024-02-21T07:59:37.747+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-21T07:59:37.759+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
